lib/manageiq/floe/workflow/runner.rb

```diff
@@ -0,0 +1,27 @@
+module ManageIQ
+  module Floe
+    class Workflow
+      class Runner
+        class << self
+          def for_resource(resource)
+            raise ArgumentError, "resource cannot be nil" if resource.nil?
+
+            scheme = resource.split("://").first
```

Comment:
Probably should use URI.parse if we always expect a URI format.

Reply:
:+1: done

Reply:
I had to revert this because:
```
>> URI.parse('docker://hello-world:latest')
/usr/lib/ruby/3.0.0/uri/rfc3986_parser.rb:67:in `split': bad URI(is not URI?): "docker://hello-world:latest" (URI::InvalidURIError)
```

Also eventually we talked about supporting `arn:...` type resources so maybe we would `split(":")` in that case